x-pack/plugins/task_manager/task_manager.ts

```diff
@@ -99,7 +99,7 @@ export class TaskManager {
           })
           .catch((err: Error) => {
             // FIXME: check the type of error to make sure it's actually an ES error
-            logger.warning(err.message);
+            logger.warning(`PollError ${err.message}`);
```

Comment:
This sort of thing should be handled by a logger tag rather than string concatenation. This ensures folks can reliably filter/drill-down alerts based on type.